src/component/form-field.js

```diff
@@ -23,7 +23,7 @@ export class FormField {
 
   attached() {
     if (!this.element.key) {
-      logger.error('an element key propery cannot be null or undefined');
+      logger.debug(`key not defined in element of type ${this.element.type} using model for value`);
```

Comment:
It is allowed to not define the key. It will use the model as the value instead of binding to a property on the model.